### ISSUE TITLE
Problem in editable popup on mobile devices

### ISSUE THREAD

**Issue by crazyursus92** [kartik-v/yii2-editable#166]
When I focus a input or textarea in my mobile phone (Android), the phone is opening keyboard and after 1 second keyboard is been closing. 

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/13435408/37143488-5b1b2c16-22cc-11e8-8502-f9bfae24ed42.gif)

**Comment by slavanb** [comment]
fix this problem
in composer.json

"kartik-v/bootstrap-popover-x": "1.4.3"

./composer.phar update

after that clean cache in /backend/web/assets  - all to trash

**Comment by phuong17889** [comment]
1.4.7 still not working
@slavanb you saved my life

**Comment by kartik-v** [comment]
Latest release is v1.7.8 and remember when you update extension and JS / CSS assets via composer you need to take care of any stale assets and cache that need to be cleared and then retry.

**Comment by alessio6181** [comment]
Hi,
I have the same problem.
This is my scenario:
`"require": {
        "php": ">=5.6.0",
        "yiisoft/yii2": "~2.0.14",
		"yiisoft/yii2-bootstrap": "@dev",
        "yiisoft/yii2-swiftmailer": "~2.0.0 || ~2.1.0",
        "rmrevin/yii2-fontawesome": "2.10.*",
        "kartik-v/yii2-tabs-x": "dev-master",
        "kartik-v/yii2-editable": "dev-master",
        "kartik-v/yii2-grid": "dev-master",
        "kartik-v/yii2-widget-touchspin": "dev-master",
		"alexandernst/yii2-device-detect": "0.0.12",
		"kartik-v/bootstrap-popover-x": "1.4.7"`

with kartik-v/bootstrap-popover-x v1.4.3 popup in editablegrid not firing.
Thanks in advance.
Alessio

**Comment by rcdevargas** [comment]
Has this been fixed? Having same trouble on mobile editable popup using v1.7.8. Please help.

**Comment by alessio6181** [comment]
> Has this been fixed? Having same trouble on mobile editable popup using v1.7.8. Please help.

no, then I used temporarily  a modal popup

**Comment by drew1two** [comment]
Hi Alessio,
Could you show me how you implemented your temporary modal popup please? 
I'm having the same issue as above.
Downgrading to 1.4.3 didn't help either.

Thanks in advance

**Comment by phuong17889** [comment]
I'm using mobile-detect and change `asPopover` to false if androidOS detected
https://github.com/serbanghita/Mobile-Detect/wiki/Code-examples
```
$os = new Mobile_Detect();
...
'asPopover' => !$os->isAndroidOS()
```

**Comment by kartik-v** [comment]
This will be fixed via kartik-v/bootstrap-popover-x#46

**Comment by kartik-v** [comment]
Fixed via [this commit](https://github.com/kartik-v/bootstrap-popover-x/commit/ded123d4ad309934435dd762b509e15d47aff748)
